### PR TITLE
[RFR][NOTEST] Removed case-sensitivity of webdriver option.

### PR DESCRIPTION
**Pull request by sbulage**

### CODE REVIEW

File: cfme/utils/browser.py [hunk 1]
```diff
@@ -258,7 +258,7 @@ def from_conf(cls, browser_conf):
                 browser_kwargs['desired_capabilities']['marionette'] = False
             return cls(WharfFactory(webdriver_class, browser_kwargs, wharf))
         else:
-            if webdriver_name == "Remote":
```
Comment (quarckster) [inline cfme/utils/browser.py]: **REQUIRED**
Please describe how did you face this issue?

Comment (sbulage) [reply cfme/utils/browser.py]: While explaining environment to Summer interns, couldn't open the browser in Selenium server. Later found that I have made typo at `webdriver_name` in the environment so I wanted to incorporate these changes in our own framework.

Comment (sbulage) [reply cfme/utils/browser.py]: I will update it accordingly.

Comment (izapolsk) [reply cfme/utils/browser.py]: I'm doubtful of necessity of this feature. Possible values of that field are Firefox, Chrome and etc. Are you going to make them case insensitive as well ?
In addition, my memory reminds me that we pass this value to webdriver itself and I'm not entirely sure it supports lower case value.

Comment (sbulage) [reply cfme/utils/browser.py]: @izapolsk I have tried with `Firefox` and `Chrome` in place of `Remote`. Able to open the browser in Selenium server as well as outside of it.
I am only making `Remote` `webdriver_name` to lower case.

**Pull request merged**